src/content/docs/iast/troubleshooting/unresponsive-or-crashing-application.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Unresponsive or crashed application
+tags:
+    - IAST
+    - New Relic IAST
+    - Troubleshooting
+metaDescription: Troubleshooting unreponsive or crashed application.
+freshnessValidatedDate: 2024-07-16
+---
+
+## Problem
+
+After instrumenting NewRelic security agent to your application, it becomes unresponsive and the application crashes.
```

Comment:
NewRelic --> New Relic

Wouldn't be more correct to say, "After enabling IAST for your APM agent, your application becomes unresponsive..."